### PR TITLE
Write section on privacy law compliance.

### PR DESCRIPTION
**Pull request by Bren2010**

### CODE REVIEW

**Review by OR13 — APPROVED** [review]
This seems to make some assumptions about the logs construction that perhaps are beyond the architecture.

I find myself wishing for a simple concrete example to look at, to be assured of these properties.

**Review by felixlinker — COMMENTED** [review]
My sentiment is the same as for #10, but I am a bit more positive here. My understanding is that this PR shows how deletion requests can be supported in a KT system using the standard construction, i.e., you show that it is possible to implement the requirement of "entry deletion must be possible."

Nevertheless, reading what you wrote, I started thinking: "But will that actually work?" Couldn't the mechanism as described here lead to equivocation attacks where a provider tells some clients that an entry was deleted and others that it was not? Maybe we require a tombstoning mechanism (as in #10)?

Again, I'm not saying that my worries should be addressed in this PR, but I think it would be better to work on a list of requirements that a KT design must provide and then proceed to the design.

**Comment by Bren2010** [comment]
> Again, I'm not saying that my worries should be addressed in this PR, but I think it would be better to work on a list of requirements that a KT design must provide and then proceed to the design.

This PR does go into the construction more than would be ideal, but solely for explanatory reasons. If we asserted that entry deletion must be possible without discussing how we plan to do that, people would jump too quickly to "this is impossible"

**Review by felixlinker — APPROVED** [review]

File: draft-ietf-keytrans-architecture.md [hunk 1]
```diff
@@ -518,6 +518,76 @@ keys and values and their modification history. It also includes traffic
 patterns, such as how often a specific key is looked up.
 
 
+# Privacy Law Compliance
```
Comment (thibmeu) [inline draft-ietf-keytrans-architecture.md:521]:
```suggestion
# Privacy Law Consideration
```
compliance may be too much

**Review by thibmeu — APPROVED** [review]
I concur with previous reviews that the example might be too much for an architecture doc, but seems scoped enough to ease comprehension, without suggesting a specific implementation.

**Pull request merged**